scene/feature-layer-rendering-mode-scene/README.metadata.json

```diff
@@ -1,35 +1,35 @@
 {
     "category": "Scene",
-    "description": "Render features statically or dynamically in 3D.",
+    "description": "Render features in a scene statically or dynamically by setting the feature layer rendering mode.",
     "ignore": false,
     "images": [
         "FeatureLayerRenderingModeScene.gif"
     ],
     "keywords": [
+        "3D",
+        "dynamic",
+        "feature layer",
+        "features",
+        "rendering",
+        "static",
         "ArcGISScene",
-        "Camera",
         "FeatureLayer",
         "FeatureLayer.RenderingMode",
         "LoadSettings",
-        "Point",
-        "Polyline",
-        "Polygon",
-        "ServiceFeatureTable"
+        "SceneView"
+    ],
+    "redirect_from": [
+        "/java/latest/sample-code/feature-layer-rendering-mode-scene-.htm"
```

Comment:
not sure this is meant to have an additional "-" after scene

Reply:
Not sure why this happened either. Looks like this was there previously (it's only flagging as a diff because the script re-writes the whole file)